roles/download/defaults/main.yml

```diff
@@ -49,7 +49,7 @@ vault_download_url: "https://releases.hashicorp.com/vault/{{ vault_version }}/va
 
 # Checksums
 istioctl_checksum: fd703063c540b8c0ab943f478c05ab257d88ae27224c746a27d0526ddbf7c370
-kubeadm_checksum: 312aeca9f56605e5d117ef901a2d8bceb701cca9662017ceb362c0d1aa91e13a
+kubeadm_checksum: 560b44a2b91747f4fb64ac8754fcf65db9a39a84c6b54d4e6483400ac6c674fc 
```

Comment:
You have a trailing space here that's failing the yaml linter

Reply:
Thanks